eth2/utils/ssz_derive/tests/tests.rs

```diff
@@ -1,22 +0,0 @@
-use ssz::Encode;
```

Comment:
Why not keep this test?

Reply:
It creates a cyclic dependency, which means we can't upload it crates.io.

See: https://github.com/rust-lang/cargo/issues/4242

(Note the issue number!)

Reply:
Let me know if you're OK with this :)

Reply:
Oh yeah, I'm fine with this. I just got confused about how `ssz_derive` doesn't already depend on `ssz`, but I get it now (the derive macro just knows about syntax).